src/pipx/commands/common.py

```diff
@@ -116,6 +159,35 @@ def _symlink_package_apps(
             )
 
 
+def find_selected_venvs_for_package(venv_container: VenvContainer, package: str, *, verbose=False) -> List[Venv]:
```

Comment:
This is used to guard against selecting multiple packages with suffixes at the same time, as well as trying to install a package with no suffix when the suffixed version is selected.